docs/Platform-Architecture/permissions-model.md

```diff
@@ -0,0 +1,164 @@
+
+- Delivery Team Member are assigned to `Delivery AAG-Users-ADP-{programme}-{devlivey project}_TeamMember` AAD group.
+- Technical Team Member are assigned to `AAG-Users-ADP-{programme}-{devlivey project}_TechUser` AAD group.
+- Delivery Team Admin are assigned to `AAG-Users-ADP-{programme}-{devlivey project}_Admin` AAD group.
+- Delivery Programme Admin are assigned to `AAG-Users-ADP-{programme}_Admin` AAD group.
+- ADP Admins are assigned to `AAG-User-ADP-PlatformEngineers` AAD group.
+
+By being added to these groups in Azure AD via the ADP Portal, users will be granted the permissions for their role in the ADP Portal.
+
+The permissions for each role in the ADP Portal are detailed below.
+
+### Platform User
+
+ADP Portal Permissions for the Platform User role:
+
+- Access to the ADP Portal.
+- Can be selected as a Delivery Project team member/admin or Delivery Programme Admin.
+- Read access to all ALBs, delivery projects, programmes, etc.
+
+### Delivery Project: Team Member
+
+ADP Portal Permissions for the Delivery Project Team Member role:
+
+- Includes all Platform User permissions.
+- Displayed as a Member of assigned Delivery Project teams.
+
+### Delivery Project: Technical Team Member
+
+ADP Portal Permissions for the Technical Team Member role:
+
+- Includes all Delivery Project Team Member permissions.
+- Scaffold/create new services for their delivery project (inc. repos).
+
+### Delivery Project: Team Admin
+
+ADP Portal Permissions for the Delivery Team Admin role:
+
+- Includes all Delivery Team Member permissions.
+- Has the ability to invite/add users to their Delivery Project team as: Team members, Technical Team Members, Team Admins, and Technical Team Admins via the ADP Portal. The function adding an team member will add them to required GitHub team, Azure DevOps project, and Azure AAD groups required for Azure resource access, depending on new team members roles in the Delivery Project.
+- Edit delivery project details in the ADP Portal.
+
+### Delivery Programme Admin
+
+ADP Portal Permissions for the Delivery Programme Admin role:
+
+- Includes all Delivery Team Admin permissions for all Delivery Projects in the programme.
+- Can create new Delivery Projects in the programme.
+- Can edit programme details for programmes they administrator in the ADP Portal.
+- Can invite/add other admins to the programmes they administer.
+
+
+### ADP Admin
+
+ADP Portal Permissions for the ADP Admin role:
+- Full access to the ADP Portal and is admin for all ALBs, delivery projects, programmes, etc.
+
+
+## GitHub Permissions
+
+GitHub Permissions are assigned and managed using GitHub teams. The following GitHub teams are automatically assigned to each repository owned by a Delivery Project:
+
+- `ADP-{programme}-{delivery project}-Contributors` GitHub team: contains all Delivery Project Technical Team Members
```

Comment:
We need to change this. The github team name should be the team name as provided exactly in GitHib.